packages/@aws-cdk/aws-ecs/lib/external/external-task-definition.ts

```diff
@@ -59,20 +65,27 @@ export class ExternalTaskDefinition extends TaskDefinition implements IExternalT
     return new ImportedTaskDefinition(scope, id, {
       taskDefinitionArn: attrs.taskDefinitionArn,
       compatibility: Compatibility.EXTERNAL,
-      networkMode: NetworkMode.BRIDGE,
+      networkMode: attrs.networkMode,
       taskRole: attrs.taskRole,
     });
   }
 
+  /**
+   * The networking mode to use for the containers in the task.
+   */
+  public readonly networkMode: NetworkMode;
+
```

Comment:
This public property is not used anywhere, so let's not add it until there is a requirement for it.

Reply:
Actually this already exists in the base class, TaskDefinition. It should still be removed from here.